pyproject.toml

```diff
@@ -66,3 +66,4 @@ ignore = ["D203", # conflict with D211
           "D213", # conflict with D212
           "D400", "D415", "ANN201", # Unnecessary
           ]
+exclude = ["src/garnet/helpers/__init__.py", "src/garnet/helpers/ui_elements/__init__.py"]
```

Comment:
I think we do not need to exclude this

Reply:
we can exclude notebooks and scripts, though

Reply:
The conda package fails to build if duplicate files are found. It usually ignores `__init__.py` files but because our linter is setup to require every file to have a docstring at the top of the file it thinks those are duplicate files.